### ISSUE TITLE
Temporarily move dragged items to back of ItemsList

### ISSUE THREAD

**Issue by g4th** [falltergeist/falltergeist#458]
When an item is being dragged from an ItemsList, it should temporarily be moved to the back of the list so that other items in the list aren't drawn over it and if it is dropped replace it back in it's original position in the list

(or move it to the back of the list when it's dropped, whichever way the game originally did it)
![dragitem](https://cloud.githubusercontent.com/assets/17906707/13899952/d22e0130-edd0-11e5-9843-e91235f86589.png)

**Comment by isage** [comment]
Yep, that's known issue. Slot items also overlap dragged item sometimes.

**Comment by isage** [comment]
It looks like
1) items are added to the start of list on drop
2) sometimes items leave empty space in list while dragged, but sometimes the leave a copy of themselves. I'm pretty sure latter is a bug in original engine and we should just leave empty space, while item is dragged.

**Comment by dithillobothrium** [comment]
In my fork/branch I fixed it. You can test
https://github.com/falltergeist/falltergeist/commit/d34abb18fbb9d19b5f377225cd4454b5588acd50
